src/meshapi/models/member.py

```diff
@@ -41,6 +41,7 @@ class Member(models.Model):
         "import process and original spreadsheet data. However this structure can be changed by admins at any "
         "time and should not be relied on by automated systems. ",
     )
+    payment_preference = models.CharField(choices=('cash', 'stripe', None), default=None)
```

Comment:
Do you mind creating a `models.TextChoices` class for this (similar to what's used in [node.py](https://github.com/nycmeshnet/meshdb/blob/main/src/meshapi/models/node.py#L17-L20)?)